Kernel/AspectSymfonyKernel.php

```diff
@@ -37,7 +38,7 @@ protected function configureAop(AspectContainer $container)
     public function init(array $options = [])
     {
         // it is a quick way to check if loader was enabled
-        $wasDebugEnabled = class_exists(DebugClassLoader::class, false);
+        $wasDebugEnabled = class_exists('\Symfony\Component\Debug\DebugClassLoader', false);
```

Comment:
What was the reason to change this syntax to the old one without `::class` accessor?